### PR TITLE
Add missing parameter

### PR DESCRIPTION
**Pull request by philtay**
Implements #115

If it's ok tests and docs will be added.

### CODE REVIEW

File: marshmallow/fields.py [hunk 1]
```diff
@@ -371,7 +374,7 @@ class Field(FieldABC):
 
     def __init__(self, default=None, attribute=None, load_from=None, error=None,
                  validate=None, required=False, allow_none=False, load_only=False,
-                 dump_only=False, **metadata):
+                 dump_only=False, missing=None, **metadata):
```
Comment (sloria) [inline marshmallow/fields.py]: `None` should be a valid value for `missing`. Perhaps you can make the default `null` here to allow `None` to be used as a value for missing data.

**Comment by philtay** [comment]
- Used null instead of none
- Added docs
- Added tests

P.S. Have you an idea on when you'll have the time to release 2.0?

**Comment by sloria** [comment]
Thanks @philtay.

I don't have a scheduled release date for 2.0, but I'm hoping to have 2.0-a released within the next 2-3 weeks.

**Comment by philtay** [comment]
@sloria don't forget to update the changelog with this new parameter

**Pull request merged**